Source/ReturnsExtensions.cs

```diff
@@ -22,10 +22,23 @@ public static class ReturnsExtensions
             return mock.Returns(tcs.Task);
         }
 
-        /// <summary>
-        /// Allows to specify the exception thrown by an asynchronous method.
-        /// </summary>
-        public static IReturnsResult<TMock> ThrowsAsync<TMock, TResult>(this IReturns<TMock, Task<TResult>> mock, Exception exception) where TMock : class
+		/// <summary>
+		/// Allows to specify the return value of an asynchronous method.
+		/// </summary>
+		public static IReturnsResult<TMock> ReturnsAsync<TMock, TResult>(this IReturns<TMock, Task<TResult>> mock, Func<TResult> value) where TMock : class
+		{
+			return mock.Returns(() =>
+			{
+				var tcs = new TaskCompletionSource<TResult>();
```

Comment:
That works as well (at least it passes the tests).  I was just trying to be consistent with the method used in the extension methods that were already there.  Not sure if there is an advantage/disadvantage either way.